Skybolt dev

Aeronautical-Eng opened the pull request:
Hi Richard,

I am ready to merge my updates of the Swift FR5. I think this is a good point to test if you are happy with my initial development methods. I have quite a bit of work to do before the next merge, so that will take a bit of time before the development is ready for a merge again.

The updates are as follows:
-The updated fuel and payload menu (This includes all possible fuel tanks and payload loads as per the Swift's FR5 pilot's notes)
-The aircraft help menu giving a quick reference to keyboard shortcuts and handy V speeds etc. 
-A preliminary to the checklists and tutorials
-The walker menu  

Regards
-Taeke

Aeronautical-Eng committed:
Development goals added

Aeronautical-Eng committed:
- add belly tank
- change tank capacities from imperial gallons to us gallons

Aeronautical-Eng committed:
- add include to SwiftFR5-main.xml
- add Belly "drop tank" and update tank capacity to us gallons
- set walker exiting position

Aeronautical-Eng committed:
- add include to SwiftFR5-main.xml
- add Belly "drop tank" and update tank capacity to us gallons
- set walker exiting position
- set pilot initial view head eye position

Aeronautical-Eng committed:
- add additional view customization settings

Aeronautical-Eng committed:
- add walker
- add additional pilot functionality (development in progress)
- add walker settings
- add save properties when shutting down simulator
- add pax for systems functionality (payload and weight calculator)
- add save menu
- add payload pilot etc.
- add limits/mass and balance (development in progress)
- add save functionality

Aeronautical-Eng committed:
-add aircraft help menu for easy access to key shortcuts and V speeds etc.

Aeronautical-Eng committed:
-add checklists (work in progress, currently c172p)

Aeronautical-Eng committed:
-add Tutorials (Currently 1% progress, still based off c172p)

Aeronautical-Eng committed:
-add additional aircraft dialog menus (development in progress)

Aeronautical-Eng committed:
-add add visual human in cockpit with walker(development in progress)

Aeronautical-Eng committed:
-add pilot systems

Aeronautical-Eng committed:
-added selectable tank and payload settings menu

Aeronautical-Eng committed:
-Add fuel and payload setup menu with mass moment arms

Aeronautical-Eng committed:
-Add payload options

Aeronautical-Eng committed:
- Add bombs and rockets and finalize fuel and payload menu

Aeronautical-Eng committed:
- update swift menu options

Aeronautical-Eng committed:
- Add bombs and rockets and finalize fuel and payload menu and cg

Aeronautical-Eng committed:
- update swift option menus (In Progress)

Aeronautical-Eng committed:
-added functionality for baggage and equipment options (In progress)

Aeronautical-Eng committed:
-add future selections in swift menu (In progress)

Aeronautical-Eng committed:
Progress status update

Aeronautical-Eng committed:
Progress status

Aeronautical-Eng committed:
-Debug electrical state on HSI

Aeronautical-Eng committed:
-Debug electrical state on HSI

Aeronautical-Eng committed:
-file not necessary at the moment

Aeronautical-Eng committed:
-Pilot still to be worked on

Aeronautical-Eng committed:
-Swift menu work in progress

Aeronautical-Eng committed:
Progress status

Aeronautical-Eng committed:
-Real manual FR5 pilots notes

Aeronautical-Eng committed:
-Swift Drawings and schematics

Aeronautical-Eng committed:
-Engine Start checklist added

Aeronautical-Eng committed:
Update panel

Aeronautical-Eng committed:
-Add stations for drop tank and separate rocket selection

Aeronautical-Eng committed:
-Corrected drop tank fuel and payload menu bug

Zaretto commented:
Thanks for your work this far. I didn't review the checklists.

1. Fuel and payload menu looks good
2. help menu is good
3, walker is good

Things I'm not sure about
1. not sure why we need "registration" dialog as this is usually baked into the livery and otherwise it requires OSG text or canvas to make the registration appear on the aircraft. Suggest removing this.

3. options dialog: show human models - generally better to use a range animation to show these when the range is greater than a meter. 
4. options dialog: drop tank should be entirely in the payload dialog; otherwise confusing to have in two places.

Aeronautical-Eng committed:
- Registration removed
- Options dialog removed until future pilot range animation development
- Options dialog for droptank only selectable under payloads

Aeronautical-Eng committed:
Delete gui/dialogs/immat.xml

No longer needed

Aeronautical-Eng committed:
Delete gui/dialogs/aircraft-dialog.xml

No longer needed

Aeronautical-Eng commented:
Hi Richard,

Thank you, your comments make a lot of sense, I have updated as per your suggestions.

Aeronautical-Eng committed:
Update SwiftFR5-main.xml

- removed registration

Aeronautical-Eng committed:
Delete Docs/Other Documentation directory

Check for public licenses or rather use links where available on websites
